swift/files/liberty/proxy-server.conf

```diff
@@ -0,0 +1,142 @@
+{% from "swift/map.jinja" import proxy with context %}
+
+[DEFAULT]
+bind_ip = {{ proxy.bind.address }}
+bind_port = {{ proxy.bind.port }}
+swift_dir = /etc/swift
+user = swift
+
+workers = {{ proxy.workers }}
+log_level = {{ proxy.get('log_level', 'INFO') }}
+
+#eventlet_debug = true
+#conn_timeout=3
+
+
+[pipeline:main]
+pipeline = catch_errors gatekeeper healthcheck proxy-logging cache {{ " ".join(proxy.get('pipeline',[])) }} container_sync bulk ratelimit {% if proxy.tempauth.enabled %}tempauth {% endif %} {%if proxy.identity is defined %}authtoken keystoneauth {% endif %}list-endpoints container-quotas account-quotas slo dlo versioned_writes proxy-logging proxy-server
+
+[app:proxy-server]
+#allow_account_management = true
+account_autocreate = true
+conn_timeout = 20
+node_timeout = 120
+use = egg:swift#proxy
+{%- if proxy.sorting_method is defined or proxy.affinity.enabled %}
+sorting_method = {{ proxy.get('sorting_method', 'affinity') }}
+{%- endif %}
+{%- if proxy.affinity.enabled %}
+write_affinity_node_count = 2 * replicas
+read_affinity = {{ proxy.affinity.read }}
+write_affinity = {{ proxy.affinity.write }}
+{%- endif %}
+
+
+{%- if proxy.tempauth.enabled %}
+[filter:tempauth]
+use = egg:swift#tempauth
+#reseller_prefix = TEMPAUTH
+
+#user_<account>_<user> = <password> [group] [group] [...] [storage_url]
+{%- for user in proxy.tempauth.users %}
+user_{{ user.account }}_{{ user.name }} = {{ user.password }} {{ user.get('groups', '') }} {{ user.get('storage_url', '') }}
+{%- endfor %}
+{%- endif %}
+
+[filter:crossdomain]
+use = egg:swift#crossdomain
+
+[filter:list-endpoints]
+use = egg:swift#list_endpoints
+
+{%- if proxy.identity is defined %}
+[filter:authtoken]
+paste.filter_factory = keystonemiddleware.auth_token:filter_factory
+identity_uri = http://{{ proxy.identity.host }}:35357/
+auth_url = http://{{ proxy.identity.host }}:35357/
+auth_uri = http://{{ proxy.identity.host }}:5000/
```

Comment:
yes - changed in swift, swift documentation is wrong. to keep backward compatible, working even doc's says the other I kept both.  approach is to keep both for liberty, remove one any higher.